examples/tutorials/converter/BETDisk_converter.py

```diff
@@ -0,0 +1,97 @@
+import argparse
+import json
+import sys
+import textwrap
+
+parser = argparse.ArgumentParser(
+    description="This program converts a given json file with BETDisk config into an updated version",
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+)
+parser.add_argument("input", type=str, help="input json file name")
+parser.add_argument("output", type=str, help="output file name")
+args = parser.parse_args()
+if args.input.endswith(".json") != True:
+    sys.exit("incorrect input")
+
+
+# loading a config json file containing BETDisk inputs
+with open(args.input) as reader:
+    read = json.load(reader)
+    read = json.dumps(read, indent=4)
```

Comment:
you are right, operating on dicts should be better a better design overall but I initially thought it was supposed to be used only for converting input for cases provided by us
since it will be available to the users it definitely should utilise a better design like you said